### PR TITLE
Remove hard coded census date

### PR DESCRIPTION
**Pull request by petechd**
### PR Context
This change adds support for validation of multiple items on a list.
Previously, it was not possible to validate contents list when one of the items was a text with a placeholder(dictionary). 
This method  adds new behaviour based on the type of data it handles. On type error it gets "text" dictionary entry from a list.
The change is described on this trello [card](https://trello.com/c/d8k4s5QA) and requires this
[schema PR](https://github.com/ONSdigital/eq-questionnaire-schemas/pull/19) to be merged.

### CODE REVIEW

File: app/validation/validator.py [hunk 1]
```diff
@@ -1878,12 +1878,20 @@ def _validate_smart_quotes(self, json_schema):
...
+                        self._error_message(
+                            f"Found dumb quotes(s) in schema text at {pointer}"
+                        )
+                    )
+            else:
+                if quote_regex.search(schema_text):
+                    errors.append(
+                        self._error_message(
+                            f"Found dumb quotes(s) in schema text at {pointer}"
+                        )
```
Comment (JamesGardiner) [inline app/validation/validator.py]:
```python
try:
    found = quote_regex.search(schema_text.get("text"))
except AttributeError:
    found = quote_regex.search(schema_text)

if found:
    errors.append(
        self.error_message(
            f"Found dumb quotes(s) in schema text at {pointer}"
        }
    )
```

Comment (petechd) [reply app/validation/validator.py]: Replaced it, thanks!

File: tests/schemas/valid/test_placeholder_in_contents_list.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "session_timeout_in_seconds": 3,
+  "title": "Test Content Variants",
+  "theme": "default",
+  "description": "A questionnaire to test content variants and variant choices",
```
Comment (ajmaddaford) [inline tests/schemas/valid/test_placeholder_in_contents_list.json]: The title and description are incorrect.

Comment (petechd) [reply tests/schemas/valid/test_placeholder_in_contents_list.json]: I forgot about this🤦‍♂️

**Pull request merged**